app/main/lib/s3_client.py

```diff
@@ -3,6 +3,8 @@
 
 from flask import current_app as app
 
+from app.main.lib.error_log import ErrorLog
```

Comment:
I'm surprised the ErrorLog is needed to get errors to sentry,  normally it already picks up on anything sent to python error output, so I'd think app.logger.error() would work?

Reply:
Maybe its a difference in configuration but I believe Sentry only surfaces raised actual errors and not log messages?

Reply:
Maybe.  Originally I added a bunch of explicit sentry error dispatch, then removed most of it because I was seeing dupes. https://stackoverflow.com/questions/73456044/why-does-sentry-log-errors-caught-in-except-block

Reply:
Hrm - well, I'll do the same then if I see the problem manifesting here? lol